ihmc-ros2-library/src/main/java/us/ihmc/ros2/ROS2QosProfile.java

```diff
@@ -16,6 +18,11 @@
  */
 public class ROS2QosProfile
 {
+   public static ROS2QosProfile DEFAULT()
+   {
+      return BEST_EFFORT();
```

Comment:
DEFAULT is now BEST_EFFORT

Reply:
Would it make sense to remove DEFAULT entirely? I think maybe it's better if you're explicit every time you use a QoS profile.

Reply:
I personally like having a `DEFAULT` although I'm not against @ds58 suggestion.
Having said nothing, I think if we're to keep a `DEFAULT` it'd be nice to have the option to change it without having to clone this repo, so either looking for an environment variable or property.

Reply:
Oof it's a conundrum. I think in any case there'd probably be a default popping up in open robotics if we don't include it here, for those wanting simple APIs and they don't care (or don't yet know they care) about what happens under the hood.

I could be convinced to remove DEFAULT though if someone feels strongly.

In any case I've added the setting from a new ROS_DEFAULT_QOS environment variable. It'll print at the beginning one of the messages:
```
240403 12:30:56:645 [INFO] (ROS2QosProfile.java:34): Topics with unspecified QoS will use RELIABLE.
```
or
```
240403 12:29:51:646 [INFO] (ROS2QosProfile.java:39): Topics with unspecified QoS will use BEST_EFFORT.
```